examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -566,17 +566,14 @@ function vertical_diffusion_boundary_layer_cache(
     else
         ts_type = TD.PhaseDry{FT}
     end
-    coef_type = SF.Coefficients{
-        FT,
-        SF.InteriorValues{FT, Tuple{FT, FT}, ts_type},
-        SF.SurfaceValues{FT, Tuple{FT, FT}, TD.PhaseEquil{FT}},
-    }
+
+    cond_type = NamedTuple{(:shf, :lhf, :E, :ρτxz, :ρτyz), NTuple{5, FT}}
```

Comment:
```suggestion
    cond_type = SF.SurfaceFluxConditions{FT <: Real}
```
We could alternatively cache the surface conditions, like this